src/ApiConsole.js

```diff
@@ -303,6 +305,7 @@ export class ApiConsole extends AmfHelperMixin(LitElement) {
       ?compatibility="${compatibility}"
       ?outlined="${outlined}"
       ?noServerSelector="${noServerSelector}"
+      ?noCustomServer="${noCustomServer}"
```

Comment:
Just a question to discuss. Shouldn't we always set noCustomServer to the selector in api-documentation? Should we allow users to modify the baseURI in the api-documentation component?

Reply:
IMHO: I don't see the down side of allowing this option to the user, for whatever reason they have to do it. I can't think of a use case right now where you would need to do it, but I also don't feel like the cons of having the option enabled are strong enough.

@jarrodek @TwoPlusTwoOne what do you think?

Reply:
First of, `noCustomServer` is nowhere declared. It's not a getter/setter nor the properties object entry.
Secondly, API Console must decide when not to render the selector in API Documentation component. And I mean not to render and not to show it. 
There are the following cases to consider:
- narrow view: documentation and request panel is never rendered at the same time so there's no issue here with children distribution
- regular (desktop) view: like in the narrow view, documentation is rendered separately to the request panel, no action needed
- wide view: the request panel is rendered next to the documentation view.

In this last case you need hide the selector in the documentation component. I don't see that logic here.

Reply:
@jarrodek 

> First of, noCustomServer is nowhere declared. It's not a getter/setter nor the properties object entry.
it was already defined by a previous change

> Secondly, API Console must decide when not to render the selector in API Documentation component. And I mean not to render and not to show it.

Got it, wide view needs to hide the selector from api-documentation.